hollow/src/main/java/com/netflix/hollow/api/codegen/indexes/HollowUniqueKeyIndexGenerator.java

```diff
@@ -81,7 +81,7 @@ protected void genConstructors(StringBuilder builder) {
 
     protected void genSimpleConstructor(StringBuilder builder) {
         builder.append("    public " + className + "(HollowConsumer consumer) {\n");
-        builder.append("        this(consumer, false);");
+        builder.append("        this(consumer, true);");
```

Comment:
Close ;)

Should neither be hardcoded to `false` nor `true`, but concatenate `isAutoListenToDataRefresh`.